src/util/cmdlineargs.h

```diff
@@ -67,16 +68,20 @@ class CmdlineArgs {
 
   private:
     CmdlineArgs() :
-        m_startInFullscreen(false), //Initialize vars
+        m_startInFullscreen(false), // Initialize vars
         m_midiDebug(false),
         m_developer(false),
         m_safeMode(false),
+#ifdef __WINDOWS__
+        m_settingsPath(QDesktopServices::storageLocation(QDesktopServices::DataLocation).append("/")) {
```

Comment:
Sorry, I mean Library/Application Support/Mixxx is the right place to put our files and it's what storageLocation(QDataLocation) returns.